src/napari_blob_detection/_widget.py

```diff
@@ -15,30 +15,53 @@ class Dimensionality(Enum):
 
 class Algorithm(Enum):
     DOG = auto()
-    DOH = auto()
     LOG = auto()
 
 
 ALGORITHM_LOOKUP = {
     Algorithm.DOG: blob_dog,
-    Algorithm.DOH: blob_doh,
     Algorithm.LOG: blob_log,
 }
 
 
 def detect_blobs(
     image: "napari.layers.Image",
     min_sigma: float = 1,
-    max_sigma: float = 50,
-    threshold: float = 0.5,
+    max_sigma: float = 10,
+    threshold: float = 0.1,
     dimensionality: Dimensionality = Dimensionality.TWO_D,
     algorithm: Algorithm = Algorithm.DOG,
 ) -> "napari.types.LayerDataTuple":
+
+    """
```

Comment:
```suggestion
    """ Detects features points on an image layer.
```

Always add a short (ideally one-line) summary of a function to a docstring.